fundingmanager.go

```diff
@@ -1463,7 +1579,7 @@ func (f *fundingManager) sendChannelAnnouncement(completeChan *channeldb.OpenCha
 		*shortChanID, chanID)
 	if err != nil {
 		fndgLog.Errorf("channel announcement failed: %v", err)
```

Comment:
This error gets logged twice, both here and where the returned error value is handled. I'll let someone more familiar with this project's style weigh in, but if the error gets returned, I don't think it's appropriate to log here. There are more instances in this method and in `sendFundingLocked`.

Reply:
I think the best way to handle this is to return a new error that combines the two: 
`return fmt.Errorf("channel announcement failed: %v", err)`